account/tests/test_views.py

```diff
@@ -36,7 +37,8 @@ def test_closed(self):
             }
             response = self.client.post(reverse("account_signup"), data)
             self.assertEqual(response.status_code, 200)
-            self.assertEqual(response.template_name, "account/signup_closed.html")
+            self.assertEqual(response.template_name,
+                             "account/signup_closed.html")
```

Comment:
I am not sure if this is an editor config difference, but can you revert these style changes?